doc/releases/changelog-0.23.0.md

```diff
@@ -537,37 +552,40 @@ the `decimals` and `show_matrices` keywords are added. `qml.drawer.tape_text(tap
 
 <h3>Bug fixes</h3>
 
-* Fixes a bug where non-trainable arguments were shifted in the `NesterovMomentumOptimizer`
+* Fixed a bug arising when multiplying `Hamiltonian` objects.
+  [(#2466)](https://github.com/PennyLaneAI/pennylane/pull/2466)
+
+* Fixed a bug where non-trainable arguments were shifted in the `NesterovMomentumOptimizer`
   if a trainable argument was after it in the argument list.
   [(#2466)](https://github.com/PennyLaneAI/pennylane/pull/2466)
 
-* Fixes a bug with `@jax.jit` for grad when `diff_method="adjoint"` and `mode="backward"`.
+* Fixed a bug with `@jax.jit` for grad when `diff_method="adjoint"` and `mode="backward"`.
   [(#2460)](https://github.com/PennyLaneAI/pennylane/pull/2460)
 
-* Fixes a bug where `qml.DiagonalQubitUnitary` did not support `@jax.jit`
+* Fixed a bug where `qml.DiagonalQubitUnitary` did not support `@jax.jit`
   and `@tf.function`.
   [(#2445)](https://github.com/PennyLaneAI/pennylane/pull/2445)
 
-* Fixes a bug in the `qml.PauliRot` operation, where computing the generator was not taking into
+* Fixed a bug in the `qml.PauliRot` operation, where computing the generator was not taking into
   account the operation wires.
   [(#2442)](https://github.com/PennyLaneAI/pennylane/pull/2442)
 
-* Fixes a bug with the padding capability of `AmplitudeEmbedding` where the
+* Fixed a bug with the padding capability of `AmplitudeEmbedding` where the
   inputs are on the GPU.
   [(#2431)](https://github.com/PennyLaneAI/pennylane/pull/2431)
 
-* Fixes a bug by adding a comprehensible error message for calling `qml.probs`
+* Fixed a bug by adding a comprehensible error message for calling `qml.probs`
   without passing wires or an observable.
   [(#2438)](https://github.com/PennyLaneAI/pennylane/pull/2438)
 
 * Call `pip show` with the subprocess library to avoid outputting a common warning.
   [(#2422)](https://github.com/PennyLaneAI/pennylane/pull/2422)
```

Comment:
The wording on this one doesn't make sense?
```suggestion
* The behaviour of `qml.about()` was modified to avoid warnings being emitted
  due to legacy behaviour of `pip`.
  [(#2422)](https://github.com/PennyLaneAI/pennylane/pull/2422)
```